internal/controllers/router/tests/create-minimal/01-assert.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+commands:
+- script: "! kubectl get router create-minimal --namespace $NAMESPACE"
```

Comment:
See what I've done in https://github.com/k-orc/openstack-resource-controller/pull/279/files#diff-44657cf22680df91f11612e8c3e4e1aa83e0c564a3ca19ddf6b09cc72d1be813

We try to delete the secret, then check that it's flagged for deletion, while still being present.